students/16th/team1/jinsookim/posting/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from user.models import Users
+# Create your models here.
+# 게시물 등록
+class Post_register(models.Model):
+    user         = models.ForeignKey('user.Users', on_delete = models.CASCADE)
+    image_url    = models.CharField(max_length=100)
+    create_time  =  models.DateField(auto_now=True)
+    upadte_time  =  models.DateField(auto_now_add=True)
```

Comment:
```suggestion
    create_time = models.DateField(auto_now_add = True)
    update_time = models.DateField(auto_now = True)
```
생성시간은 auto_now_add로 만들어야되고, 수정한 시간은 auto_now로 만들어서 저장 해줘야 되요